Automattic/simplenote-ios
# Experimental Scheme

### Details
In this PR we're:

1. Adding a new scheme, to be known as **Simplenote Experimental**
2. Wiring custom GAE and Simperium Endpoints, when appropriate
3. Wiring a new Alert, to be presented whenever an Experimental build is being executed
4. Anything built with the **Internal** Target will have the `Nü Simplenote` Bundle Display Name

This PR is based on @mokagio's work, with few tiny tweaks here and there.
cc @eshurakov 

### Test
1. Select the **Simplenote Scheme**
2. Fresh install the app
3. Run a debug build as usual

- [x] Verify no Alert shows up while you're in the Onboarding UI
- [x] Verify that you can log into your account / sync your notes

### Test
1. Select the **Simplenote Experimental Scheme**
2. Fresh install the app
3. Run the app (the target should be **Distribution Internal** )

- [x] Verify an Alert shows up as you launch the app
- [x] Verify that you can log into your account / sync your notes

### Release
These changes do not require release notes.
